### DISCUSSION TITLE
Motion control mappings

### DISCUSSION THREAD

**Discussion by wtfbbqhax** [darthcloud/BlueRetro#478] [Ideas]
Hi everyone, I've been testing my device all weekend and one idea i would like to try is mapping a motion control inputs from wii motes + nunchucks, and switch joy cons to button presses and/or combos. 

Presented as a user story, 

As a player I would like to try playing Zelda64 with motion controls mapped similarly to Twilight Princess, and Skyward Sword. 

Happy to help, if I can contribute my time I will!

**Comment by darthcloud** [comment]
This is possible but it's not something I'm interested personally so it would have to be contributed by someone else.
